compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmSymbols.kt

```diff
@@ -60,6 +60,7 @@ class JvmSymbols(
     private val kotlinReflectPackage: IrPackageFragment = createPackage(FqName("kotlin.reflect"))
     private val javaLangPackage: IrPackageFragment = createPackage(FqName("java.lang"))
     private val javaLangInvokePackage: IrPackageFragment = createPackage(FqName("java.lang.invoke"))
+    private val javaLangReflectPackage: IrPackageFragment = createPackage(FqName("java.lang.reflect"))
```

Comment:
I wonder if it makes sense to extract `java.lang.reflect` API to a separate class ~`JvmReflectSymbols` and create it on-demand in `ReflectiveAccessLowering`. It'd be useful both for readability (`JvmSymbols` is very big and slow to analyze already) and performance (creating numerous IR declarations which won't be used in 99% of scenarios seems unnecessary).

But it's not a blocker for this review. We can think about it separately, together with maybe extracting some other groups of symbols out of this class with the same purpose.

Reply:
Makes sense to me. I will push a separate PR with a start on this, then we can iterate on that there.

OK?

Reply:
Yeah, thanks, I think it can be postponed